common/app/templates/headerInlineJS/enableStylesheets.scala.js

```diff
@@ -21,6 +21,47 @@
 (function (styleSheetLinks, documentStyleSheets) {
 
     // Check the stylesheet has downloaded, then set media to `screen`. If not, go again.
+    @if(mvt.LoadCSSRafTest.isParticipating) {
+        // http://paulirish.com/2011/requestanimationframe-for-smart-animating/
+        // http://my.opera.com/emoller/blog/2011/12/20/requestanimationframe-for-smart-er-animating
+        // requestAnimationFrame polyfill by Erik Möller. fixes from Paul Irish and Tino Zijdel
+        // MIT license
+        (function () {
+            var lastTime = 0;
+            var vendors = ['moz', 'webkit'];
+            for(var x = 0; x < vendors.length && !window.requestAnimationFrame; ++x) {
+                window.requestAnimationFrame = window[vendors[x]+'RequestAnimationFrame'];
+                window.cancelAnimationFrame = window[vendors[x]+'CancelAnimationFrame'] || window[vendors[x]+'CancelRequestAnimationFrame'];
+            }
+
+            if (!window.requestAnimationFrame)
+                window.requestAnimationFrame = function(callback, element) {
+                    var currTime = new Date().getTime();
+                    var timeToCall = Math.max(0, 16 - (currTime - lastTime));
+                    var id = window.setTimeout(function() { callback(currTime + timeToCall); },
+                      timeToCall);
+                    lastTime = currTime + timeToCall;
+                    return id;
+                };
```

Comment:
that's only a massive precaution, it's [almost not needed](http://caniuse.com/#feat=requestanimationframe) (on mobile anway)